src/Dumplie/Application/Query/Customer/Result/CartItem.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Query\Customer\Result;
+
+final class CartItem
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var int
+     */
+    private $quantity;
+
+    /**
+     * @var float
+     */
+    private $price;
+
+    /**
+     * @var string
+     */
+    private $currency;
+
+    /**
+     * CartItem constructor.
+     * @param string $sku
+     * @param int $quantity
+     * @param float $price
+     * @param string $currency
+     */
+    public function __construct(string $sku, int $quantity, float $price, string $currency)
+    {
+        $this->sku = $sku;
+        $this->quantity = $quantity;
+        $this->price = $price;
```

Comment:
Jednym z głównych założeń a w zasadzie kompromisów jest brak obsługi zamówień złożonych z wielu walut. Co defacto wymusza w sklepie jedną walutę (dodatkowe mogą być poglądowe w stylu "Ile by to kosztowało w Euro").